Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM docker.io/bitnami/keycloak:24.0.5
+
+CMD ["kc.sh", "start-dev"]
```

Comment:
Does `kc.sh` mean `keycloak.sh`? Is this some sort of standard I don't know?

Why is the command `start-dev`? I assume this is for local dev testing and not for publishing an image?

Reply:
is this Dockerfile used? I don't see the reference on the docker-compose file